koans/about_asserts.rb

```diff
@@ -13,28 +13,28 @@ def test_assert_truth
   # Enlightenment may be more easily achieved with appropriate
   # messages.
   def test_assert_with_message
-    assert false, "This should be true -- Please fix this"
+    assert true, "This should be true -- Please fix this"
```

Comment:
This should not show up as a change once PR2 is merged

Reply:
I think you may not have been working against an updated copy of the repo. A rebase would have made this cleaner